src/core/db/repository/abstract_repository.py

```diff
@@ -44,6 +44,7 @@ async def update(self, id: UUID, instance: DatabaseModel) -> DatabaseModel:
         return instance  # noqa: R504
 
     async def update_all(self, instances):

```

Comment:
```suggestion
    async def update_all(self, instances: list[DatabaseModel]) -> list[DatabaseModel]:
```
Давай добавим аннотации, чтобы все было в одном стиле)